docs/get-started/installing-cardano-node.md

```diff
@@ -89,23 +89,16 @@ You can check if `ghcup` has been installed correctly by typing `ghcup --version
 The GHCup Haskell installer, version v0.1.14.1
 ```
 
-`ghcup` will install the latest stable version of `ghc`. However, as of the time of writing this, [Input-Output](https://iohk.io) recommends using `ghc 8.10.4`. So, we will use `ghcup` to install and switch to the required version. 
-
-```bash
-ghcup install ghc 8.10.4
-ghcup set ghc 8.10.4
-```
-
 Finally, we check if we have the correct `ghc` and `cabal` versions installed.
```

Comment:
ghcup installer is preferred method to install ghc and manage different ghc versions.
Not sure why you removed this section